istio.deps

```diff
@@ -5,12 +5,5 @@
 		"repoName": "api",
 		"file": "repositories.bzl",
 		"lastStableSHA": "36b42252042c0e8b229a0d66059af184c3105f9d"
```

Comment:
You can remove this one too.
In the `release-1.3` branch it's pointing to a non-existent commit (it's actually a commit in `istio/envoy`, not `istio/api`), and nothing broke.
https://github.com/istio/proxy/blob/release-1.3/istio.deps

So the whole file could be deleted.

Reply:
You haven't triggered a release of 1.3 yet is why. It is used https://github.com/istio/istio/blob/f8295503296ec6bae1b1047cc1491469d5e72754/release/gcb/generate_manifest.sh#L77-L85

Reply:
OK, I'll fix that.